scripts/release-all

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+. scripts/library.sh
+
+RELEASE_PROGRAMS=( $( relay::cli::default_programs ) )
+
+for PROGRAM in "${RELEASE_PROGRAMS[@]}"; do
+  echo "# release (go): ${PROGRAM}"
+  scripts/release "${PROGRAM}"
```

Comment:
the sdk had "release-all" because there were multiple programs ... but whatever, i suppose it doesn't hurt.